database/sql/V017__add_full_fee.sql

```diff
@@ -0,0 +1,11 @@
+-- 1. Add full fee column.
+-- This column is a part of order metadata. Its values are used to settle orders;
+-- they are only relevant for non-expired orders. Therefore, we can set the default
+-- to `0`, deploy an intermediate version that stores this value but never actually
+-- reads it, then wait till all the old orders are expired.
+ALTER TABLE orders
+    ADD COLUMN full_fee_amount numeric(78,0) NOT NULL DEFAULT 0;
```

Comment:
For Postrges noobs like me:
https://www.postgresql.org/docs/9.1/datatype-numeric.html#DATATYPE-NUMERIC-DECIMAL

`numeric(78,0)` means a maximum of 78 (**decimal**) digits, where 0 of them are fractional. `U256::MAX.to_string().len() == 78` so it all checks out.

Reply:
I'd slightly prefer 0 so we don't lose information on which orders we don't have this for, and so they don't get confused with orders that were created without any subsidy.

I have no strong opinion though.